db/migrate/20150902210721_create_projects_view.rb

```diff
@@ -0,0 +1,51 @@
+class CreateProjectsView < ActiveRecord::Migration
+  def up
+     execute <<-SQL
+
+       CREATE OR REPLACE FUNCTION public.img_thumbnail(projects, size text)
+       RETURNS text
+       LANGUAGE sql
+       STABLE
```

Comment:
We do not, settings have security definer already, so this functions does not access the database by itself :D